rts/Lua/LuaHandleSynced.cpp

```diff
@@ -1635,15 +1635,13 @@ bool CSyncedLuaHandle::FeaturePreDamaged(
  *
  * @function SyncedCallins:ShieldPreDamaged
  *
- * If the weapon is a hitscan type (BeamLaser or LightningCanon) then proID is nil and beamEmitterWeaponNum and beamEmitterUnitID are populated instead.
- *
- * @param projectileID integer
+ * @param projectileID integer? `nil` when the weapon type is `BeamLaser` or `LightningCannon`
```

Comment:
Right now it is always an integer. For beams with no valid projectile ID it's always a magic value of -1 but it being nil would be nice: https://github.com/beyond-all-reason/RecoilEngine/issues/880